easybuild/easyconfigs/i/iccifort/iccifort-2019.4.243-GCC-8.3.0-2.32.eb

```diff
@@ -0,0 +1,42 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'iccifort'
+version = '2019.4.243'
+versionsuffix = '-GCC-8.3.0-2.32'
```

Comment:
Is there a reason why this file is included? It seems we are all in favor of dropping the versionsuffix, so this file isn't very relevant

Reply:
It's just there so people can test with current develop until foss-2019b is merged.

Reply:
@bartoldeman It's a bit confusing to have both in here. Please remove this one, and clearly mention in the PR description that this depends on `GCC-8.3.0.eb` from #8567?

Reply:
Hold on, the other easyconfig doesn't require `GCC-8.3.0.eb` at all? How exactly does this depend on #8567? It doesn't I think?

Reply:
You're right, the other easyconfig depends on `GCCcore, 8.3.0` in `dependencies`, not on `GCC`.